### PR TITLE
Fix some formatting and typos

### PR DESCRIPTION
**Pull request by jsdanielh**
- Fix some formatting and typos in several files.
- Rename `RpcUtilityMethods`'s `get_proccess_details` method to
  `get_process_details`.

### CODE REVIEW

**Comment by orweis** [comment]
Hi @jsdanielh !
Thanks for the work here. I'll try to review it all tomorrow, typo fixes look great, there are some type changes I'd need to look more deeply into than a quick skim.

File: tests/custom_methods_test.py [hunk 1]
```diff
@@ -13,7 +13,8 @@
 import sys
 
 # Add parent path to use local src as package for tests
-sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir)))
+sys.path.append(os.path.abspath(os.path.join(
```
Comment (orweis) [inline tests/custom_methods_test.py:16]: This seems inconsistent with the other changes here (preferring to have code in a single line)

File: tests/fast_api_depends_test.py [hunk 2]
```diff
@@ -5,7 +5,8 @@
 from websockets.exceptions import InvalidStatusCode
 
 # Add parent path to use local src as package for tests
-sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir)))
+sys.path.append(os.path.abspath(os.path.join(
```
Comment (orweis) [inline tests/fast_api_depends_test.py:8]: This seems inconsistent with the other changes here (preferring to have code in a single line)

File: tests/trigger_flow_test.py [hunk 3]
```diff
@@ -6,7 +6,8 @@
 import sys
 
 # Add parent path to use local src as package for tests
-sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir)))
+sys.path.append(os.path.abspath(os.path.join(
```
Comment (orweis) [inline tests/trigger_flow_test.py:9]: This seems inconsistent with the other changes here (preferring to have code in a single line)

**Review by orweis — CHANGES_REQUESTED** [review]
Looks great- few comments inline; most not critical, only the float->int change requires further discussion

**Comment by jsdanielh** [comment]
> Looks great- few comments inline; most not critical, only the float->int change requires further discussion

@orweis I applied your suggestion on the type change for the timeout. I actually went and changed it in some other places that were specifying `int`.
For the line break changes, they go more into complying with [PEP8 maximum line specification](https://peps.python.org/pep-0008/#maximum-line-length). Although I know there are some lines that doesn't fully comply with it, I thought a change towards that path would help. Let me know what you think.

**Review by orweis — APPROVED** [review]
Awesomeness. :)
Thank you @jsdanielh

**Pull request merged**